ddtrace/contrib/elasticsearch/transport.py

```diff
@@ -1,15 +1,33 @@
+import json
+
 from elasticsearch import Transport
+from elasticsearch import Urllib3HttpConnection
 
 from .quantize import quantize
 from . import metadata
 from ...compat import urlencode
-from ...ext import AppTypes
+from ...ext import AppTypes, http
 from ...util import deprecated
 
+
 DEFAULT_SERVICE = 'elasticsearch'
 SPAN_TYPE = 'elasticsearch'
 
 
+class TracedConnection(Urllib3HttpConnection):
+    """Change to elasticsearch http connector so that it
+        adds the http status code to data
+    """
+
+    def perform_request(self, method, url, params=None, body=None, timeout=None, ignore=()):
+        status, headers, data = super(TracedConnection, self).perform_request(method, url, params,
+            body, ignore=ignore, timeout=timeout)
+        data = json.loads(data)
```

Comment:
this approach of deserializing, and re-serializing json to change a single field will be slow for big payloads. we'll have to find a more performant way of doing this